### DISCUSSION TITLE
Where to find the official release of Keycloak 22.0.10 ?

### DISCUSSION THREAD

**Discussion by guptakaran27** [keycloak/keycloak#30548] [Q&A] [answered]
Hi, 
We see that 22.0.10 is a tag created on github, and many security issues are showing fixed on 22.0.10.
But this version is not showing in the official releases of Keycloak. 
Any comments/updates on this ? 

Thanks!

**Comment by jonkoops** [comment, marked as answer]
These tags are the basis for [Red Hat Brand of Keycloak (RHBK)](https://access.redhat.com/products/red-hat-build-of-keycloak/), they are LTS versions with commercial support, and are not released to the community as mainline Keycloak versions. If you want to have a secure version of Keycloak you'll have to use the latest stable version.
